validator/client/beacon-api/event_handler.go

```diff
@@ -0,0 +1,112 @@
+package beacon_api
+
+import (
+	"context"
+	"net/http"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v4/api"
+)
+
+const eventByteLimit = 512
```

Comment:
I set the limit to the first power of 2 bigger than the size of the head event, which is 446 bytes.